frontend/src/utils/time.js

```diff
@@ -0,0 +1,867 @@
+//
+// SPDX-FileCopyrightText: 2020 SAP SE or an SAP affiliate company and Gardener contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import includes from 'lodash/includes'
+import sample from 'lodash/sample'
+import join from 'lodash/join'
+import padStart from 'lodash/padStart'
+import map from 'lodash/map'
+import takeWhile from 'lodash/takeWhile'
+
+export const locations = [
```

Comment:
move locations and timezones in separate files. Create a folder for time and move this into index.

Reply:
moved to dedicated files